api/circulation.py

```diff
@@ -485,7 +485,12 @@ class BaseCirculationAPI:
     # For instance, the combination ("application/epub+zip",
     # "vnd.adobe/adept+xml") is called "ePub" in Axis 360 and 3M, but
     # is called "ebook-epub-adobe" in Overdrive.
-    delivery_mechanism_to_internal_format = {}
+    delivery_mechanism_to_internal_format: Dict[
+        Tuple[Optional[str], Optional[str]], str
+    ] = {}
+
+    def __init__(self, _db, collection):
+        pass
```

Comment:
Should this raise 'NotImplementedError' as the instance below does?

Reply:
Yeah, it makes sense. I think I first had both of them as `pass` but forgot to change this one. I'll change it 👍

Reply:
> Should this raise 'NotImplementedError' as the instance below does?

As it turned out - no. Tests uses BaseCirculationAPI class directly so this raises an error and fails those tests. I'll revert back to pass.